### PR TITLE
chore: update test/OWNERS

### PR DESCRIPTION
**Pull request by chensun**
**Description of your changes:**

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: test/OWNERS [hunk 1]
```diff
@@ -1,7 +1,10 @@
 approvers:
-  - Ark-kun
-  - IronPan
-  - chensun
```
Comment (gkcalat) [inline test/OWNERS:4]: Are you sure?

File: test/OWNERS [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+  - connor-mccarthy
+  - gkcalat
+  - jlyaoyuli
+  - Linchin
 reviewers:
-  - Ark-kun
   - chensun
```
Comment (gkcalat) [inline test/OWNERS:7]: Should we add Connor here too?

**Review by gkcalat — COMMENTED** [review]
Hi Chen! Thank you. It is probably fine to remove owners from this file that are present in the root OWNERS. Would that affect the process for assigning the suggested reviewers for new PRs?

/LGTM

File: test/OWNERS (hunk 1 above)
Comment (chensun) [reply test/OWNERS:4]: IronPan and I are OWNERS at root folder, so not really necessary to repeat us here.

File: test/OWNERS (hunk 2 above)
Comment (chensun) [reply test/OWNERS:7]: I was thinking backend and frontend tests are more likely to be affected by external changes, so probably Connor's attention are less required.

**Comment by chensun** [comment]
> Would that affect the process for assigning the suggested reviewers for new PRs?

IIUC, that's controlled by the `reviewers` list, and the reason I still keep myself in the `reviewers` list.

**Review by connor-mccarthy — COMMENTED** [review]
/lgtm

**Pull request merged**